tests/component-tests/progress/html/Progress-test.html

```diff
@@ -21,6 +21,10 @@
         <script type="text/javascript" src="../../../test-core/jqUnit/js/jqUnit.js"></script>
         <script type="text/javascript" src="../../../test-core/jqUnit/js/jqUnit-browser.js"></script>
 
+        <!-- Required to ensure we sent coverage data at the end of the test run on this page -->
+        <script src="/testem.js"></script>
```

Comment:
It would be nice to be consistent with including `type="text/javascript"` at using `<script>` tags.